wolfssl/wolfcrypt/error-crypt.h

```diff
@@ -259,8 +259,10 @@ enum {
     SM4_GCM_AUTH_E      = -298,  /* SM4-GCM Authentication check failure */
     SM4_CCM_AUTH_E      = -299,  /* SM4-CCM Authentication check failure */
 
-    WC_LAST_E           = -299,  /* Update this to indicate last error */
-    MIN_CODE_E          = -300   /* errors -101 - -299 */
+    AES_EAX_AUTH_E = -300, /* AES-EAX Authentication check failure */
```

Comment:
We have a slack thread on this topic. It might be easier to just refactor the various `*_AUTH_E` to a single error code.